.travis.yml

```diff
@@ -1,7 +1,7 @@
 node-preset: &node # Used internally to share configuration, ignored by travis.
   language: node_js
   node_js:
-    - "10"
+    - "12.22.0"
```

Comment:
```suggestion
    - "12"
```

Reply:
it's better to allow auto-upgrade by travis

Reply:
I just checked, and indeed, if I specify 12, the latest 12 will be installed. And since the latest is currently 12.22.9 it will work.